### PR TITLE
login_signup_pages --Designed

### PR DESCRIPTION
**Pull request by itsmebipul00**
Login_Signup Pages Designed

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CandyShop</title>
+    <link rel="stylesheet" href="https://nextui.netlify.app/docs.css">
+    <link rel="stylesheet" href="form.css">
+    <script defer src="https://nextui.netlify.app/docs.js"></script>
+    <script defer src="https://kit.fontawesome.com/8c1231c9b8.js" crossorigin="anonymous"></script>
+
+
```
Comment (iRohitGaur) [inline login.html]: extra spacing, can remove

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: form.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.form-login {
+  width: 40%;
+  margin: 0 auto;
+}
+.already-done {
+  margin-top: 0.5rem;
```
Comment (zoyatanweer) [inline form.css:13]: it can be in pixels.

File: signup.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CandyShop</title>
+    <link rel="stylesheet" href="https://nextui.netlify.app/docs.css">
+    <link rel="stylesheet" href="form.css">
+    <script defer src="https://nextui.netlify.app/docs.js"></script>
+    <script defer src="https://kit.fontawesome.com/8c1231c9b8.js" crossorigin="anonymous"></script>
+
```
Comment (zoyatanweer) [inline signup.html]: extra spacing here. you can remove it.

**Review by zoyatanweer — APPROVED** [review]
good to merge with suggested changes.

File: signup.html (hunk 3 above)
Comment (itsmebipul00) [reply signup.html]: Okay ma'am✌️

**Pull request merged**